sections/footer.liquid

```diff
@@ -73,14 +73,43 @@
                       {%- endfor -%}
                     </ul>
                   {%- endif -%}
+                {%- when 'brand_information' -%}
+                  <div class="footer-block-brand-information">
+                    {%- if settings.brand_image != blank -%}
+                      {%- assign brand_image_height = settings.brand_image_width | divided_by: settings.brand_image.aspect_ratio -%}
+                      <style>
+                        .brand-information__image-wrapper {
+                          max-width: min(100%, {{ settings.brand_image_width }}px);
+                        }
+                      </style>
+                      <div class="brand-information__image-wrapper global-media-settings">
+                        {{ settings.brand_image | image_url: width: 1100 | image_tag:
+                          loading: 'lazy',
+                          widths: '50, 100, 150, 200, 300, 400, 550, 800, 1100',
+                          height: brand_image_height,
+                          width: settings.brand_image_width
+                        }}
+                      </div>
+                    {%- endif -%}
+                    {%- if settings.brand_headline != blank -%}
+                      <h2 class="footer-block__heading rte">{{ settings.brand_headline }}</h2>
+                    {%- endif -%}
+                    {%- if settings.brand_description != blank -%}
+                      <div class="rte>{{ settings.brand_description }}</div>
+                    {%- endif -%}
+                    {%- if block.settings.show_social -%}
+                      {%- render 'social-icons' -%}
+                    {%- endif -%}
+                  </div>
                 {%- when 'image' -%}
                   <div class="footer-block__details-content footer-block-image {{ block.settings.alignment }}">
                     {%- if block.settings.image != blank -%}
                       {%- assign image_size_2x = block.settings.image_width | times: 2 | at_most: 5760 -%}
                       <img
                         srcset= "{{ block.settings.image | image_url: width: block.settings.image_width }}, {{ block.settings.image | image_url: width: image_size_2x }} 2x"
-                        src="{{ block.settings.image | image_url: width: 400 }}"
+                        src="{{ block.settings.image | image_url: width: 760 }}"
                         alt="{{ block.settings.image.alt | escape }}"
+                        class="global-media-settings"
```

Comment:
This image would also need a wrapping div with that class like you did for the branding block. It might have got lost in the rebases.
<details>
<summary>Screenshot</summary>

![](https://screenshot.click/09-46-03nig-ij2bj.png)
</details>

Reply:
Updated :D